src/components/Datatable/Datatable.tsx

```diff
@@ -49,6 +49,7 @@ function Datatable<D extends Record<string, unknown>>({
   isBatchModuleEnabled = true,
   controlsConfig = {},
   tableConfig = {},
+  pageButtonsCount,
```

Comment:
This should be hidden inside the `tableConfig` property as written in the ticket. This setting is coupled only with the table itself and there is no need to expose it as a separate property